presto-main/src/test/java/io/prestosql/sql/planner/assertions/ExpressionVerifier.java

```diff
@@ -415,12 +417,7 @@ protected Boolean visitInListExpression(InListExpression actual, Node expected)
         if (actuals.size() != expecteds.size()) {
             return false;
         }
-        for (int i = 0; i < actuals.size(); i++) {
-            if (!process(actuals.get(i), expecteds.get(i))) {
-                return false;
```

Comment:
this loop is a bit easier to debug as one can set a breakpoint here to see why it return false. However it is also easy to do it with streams.

Reply:
No, IMO it's not nearly as easy to debug with streams, not to mention "step into".